.travis.yml

```diff
@@ -33,7 +33,14 @@ script:
   - sudo --preserve-env=PATH,PERL5LIB ./configure.pl --batch
   - sudo --preserve-env=PATH,PERL5LIB su -m backuppc -c '/usr/local/BackupPC/bin/BackupPC -d'
   - sudo cat /var/log/BackupPC/LOG
-  - sudo egrep -v '[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} (Reading hosts file|BackupPC HEAD \(Perl v5\.[0-9]{1,2}\.[0-9]\) started, pid [0-9]+|Next wakeup is [0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2})$' /var/log/BackupPC/LOG; test $? -eq 1
+  - |
+    cat <<EOF >regex.txt
+    [0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} Reading hosts file
+    [0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} BackupPC HEAD \(Perl v5\.[0-9]{1,2}\.[0-9]\) started, pid [0-9]+
+    [0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} Next wakeup is [0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}
+    EOF
+    sudo egrep -v /var/log/BackupPC/LOG -f regex.txt
+    test $? -eq 1
```

Comment:
I am not sure if we should do this. The separate regexps in the configuration file are definitely easier to read, from one hand, but makes harder to read Travis output, on the other hand.

Before:
![изображение](https://user-images.githubusercontent.com/2275981/83541456-57d02d80-a502-11ea-8b74-48e5347b3a61.png)

After:
![изображение](https://user-images.githubusercontent.com/2275981/83541553-746c6580-a502-11ea-9e92-feac48090378.png)

Another thing that concerns me is a multiline command ( - |). Sometimes it hard to understand what happens when one of the commands fails (even with `set -e`). It could lead to false negatives.

Reply:
I get your point, I changed the setup. We should have better readability everywhere now.

Reply:
Working great, thanks!